app/models/group.js

```diff
@@ -106,7 +112,7 @@ exports.addModel = function(database) {
         .then(function(user) {
           database.hmsetAsync(mkKey(['user', that.id]),
                               { 'screenName': that.screenName,
-                                'visibility': that.visibility,
+                                'isPrivate': that.isPrivate,
```

Comment:
I wonder how did Redis save Integer value to the database? It's supposed to be string.